src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityLink.tsx

```diff
@@ -39,17 +50,20 @@ const AnnotationActivityLink = ({
 
     return (
         <PlainButton
-            className="bcs-AnnotationActivity-link"
+            className={classNames('bcs-AnnotationActivityLink', className)}
             data-testid="bcs-AnnotationActivity-link"
             isDisabled={isDisabled}
             onClick={handleClick}
             onMouseDown={handleMouseDown}
+            title={translatedMessage}
```

Comment:
I think the timestamp uses a `Tooltip` component to display the full string. Should we do the same here to stay consistent, especially since they're siblings?

Reply:
Design had said just to use the default browser tooltip. The `Tooltip` component also seems a bit aggressive -- as soon as hover occurs it popups up and in this case is exactly the same as the current text so the value add isn't very much

Reply:
I had assumed that the title was used to display an untruncated version of the string, similar to the timestamp. Is that not the case? If not, do we need the title at all?